script.js

```diff
@@ -6,7 +6,25 @@ function setup() {
 
 function makePageForEpisodes(episodeList) {
   const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+
+  return episodeList.map(episode => {
+    const div = document.createElement("div");
+    div.classList.add("episode");
+
+    const h3 = document.createElement("h3");
+    h3.textContent = `${episode.name} - ${episode.number < 10
+      ? "S0"+episode.season+"E0"+episode.number 
+      : "S0"+episode.season+"E"+episode.number}`;
+
+    const img = document.createElement("img");
+    img.src = episode.image.medium;
+
+    const p = document.createElement("p");
+    p.textContent = episode.summary;
```

Comment:
it's all good
just a little minor bug here 
if you check what is in `episode.summary` you'll see html code in there.
so instead of putting it into `p.textContent` you might want to put it into `p.innerHTML` instead